Automattic/Gravatar-SDK-Android
# QuickEditor: SelectableAvatar Composable

Closes #220 

### Description

This PR adds an internal composable to be used in the Native Avatar Picker. As we are still working on the final design, this can change a bit, but this PR also allows us to set up Screenshot testing for this new module.

<img width="137" alt="image" src="https://github.com/user-attachments/assets/65d3bb15-5fdc-4dae-a896-c9024197730f">
<img width="137" alt="image" src="https://github.com/user-attachments/assets/0dafb84e-face-4a00-ab66-5c4afd4c37e5">


### Testing Steps

- You can run the `@Preview`s in a device in other to see how they look:
<img width="137" alt="image" src="https://github.com/user-attachments/assets/80600ebc-eb80-46fe-98e3-2eafaf1dcdd6">
<img width="129" alt="image" src="https://github.com/user-attachments/assets/bc3b5e5e-6cb5-410f-bc11-e817c0020072">

- Code Review and CI
